README.md

```diff
@@ -175,7 +175,7 @@ Feel free to check the [issues page](../../issues/).
 <!-- ACKNOWLEDGEMENTS -->
 
```

Comment:
- [ ] Live demo link in your ReadMe is broken on _line 77_, kindly endeavor to fix or remove it if this app is not yet deployed. Similarly, kindly remove redundant video presentation link on line _41-43_.